.changeset/brave-kangaroos-sip.md

```diff
@@ -0,0 +1,8 @@
+---
+"@octopusdeploy/add-changeset": patch
+"@octopusdepoy/current-branch-name": patch
```

Comment:
This doesn't look quite correct? Presumably it should be 

```suggestion
"@octopusdeploy/current-branch-name": patch
```

?

Reply:
Ah! great catch woops!